CHANGELOG.md

```diff
@@ -7,228 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+  - Tab will be hidden when the `Use Token Detection` setting is disabled
+- [#11798](https://github.com/MetaMask/metamask-extension/pull/11798): Add fake token warning and replacing `Add Token` labels with `Import Tokens`
+- [#11755](https://github.com/MetaMask/metamask-extension/pull/11755): Add `Refresh list` and `import tokens` link to the main page
+  - Links will be hidden when the `Use Token Detection` setting is disabled
+- [#11851](https://github.com/MetaMask/metamask-extension/pull/11851): Add Token Detection Toggle to Security & Privacy (Settings)
+  - This feature is experimental and is off by default, when enabled, it will add auto-detected tokens to a user's asset list
+- [#12042](https://github.com/MetaMask/metamask-extension/pull/12042): Add Max Fee Per Gas to transaction breakdown
+- [#11999](https://github.com/MetaMask/metamask-extension/pull/11999): Add eth_feeHistory to API callable by dapps
+- [#11849](https://github.com/MetaMask/metamask-extension/pull/11849): Add subtitles for additional languages (Secret Recovery Phrase Video)
+- [#11772](https://github.com/MetaMask/metamask-extension/pull/11772): Add tooltip to better explain possible effects of setting gas fees below MetaMask's estimates
+
+### Changed
+- [#12019](https://github.com/MetaMask/metamask-extension/pull/12019): Show the user's address book name for a contract, instead of the contract address, when on the confirmation screen
+- [#11915](https://github.com/MetaMask/metamask-extension/pull/11915): Improved time to see Swaps quotes
+- [#11802](https://github.com/MetaMask/metamask-extension/pull/11802): Replace hardcoded sent ether label on confirmation screen
+- [#11982](https://github.com/MetaMask/metamask-extension/pull/11982): Sort addresses within letter groups in address book / contacts alphabetically
+- [#12000](https://github.com/MetaMask/metamask-extension/pull/12000): Autofilling new issue link with url and domain (Phishing detect page)
+- [#11964](https://github.com/MetaMask/metamask-extension/pull/11964): Disabled spell-check on ens input
+- [#12013](https://github.com/MetaMask/metamask-extension/pull/12013): Improving warning text for eth_sign transactions
+- [#11945](https://github.com/MetaMask/metamask-extension/pull/11945): Send user to account page upon adding a custom network
+- [#11895](https://github.com/MetaMask/metamask-extension/pull/11895): Update header on Send page from `Add Recipient` -> `Send to`
+- [#11850](https://github.com/MetaMask/metamask-extension/pull/11850): Use "Secret Recovery Phrase" text throughout the onboarding flow
+- [#11795](https://github.com/MetaMask/metamask-extension/pull/11795): Autofocus the unit field in send screen
+- [#11889](https://github.com/MetaMask/metamask-extension/pull/11889): Update gas controls copy
+- [#11805](https://github.com/MetaMask/metamask-extension/pull/11805): Update tooltip text for gas fees
```

Comment:
We could describe this better I think
```suggestion
- [#11805](https://github.com/MetaMask/metamask-extension/pull/11805): Update tooltip text for gas fees to avoid referring to custom networks as "Ethereum".
```